src/planned-tests.html

```diff
@@ -50,6 +50,43 @@ <h3 class="govuk-body govuk-!-margin-bottom-6">
       <p class="govuk-body">
         If you get an alert, your phone or tablet may make a loud siren-like sound. You do not need to take any action.
       </p>
+
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
+        Ongoing
+      </h2>
+      <h3 class="govuk-body govuk-!-margin-bottom-6">
+        UK
+      </h3>
+      <p class="govuk-body">
+        Mobile phone networks in the UK are testing emergency alerts.
+      </p>
+      <p class="govuk-body">
+        You may get an alert if you have an Android phone or tablet.
```

Comment:
I we sure iPhone users won’t get test alerts – or is it just that they cannot opt out?

Reply:
iPhone users won’t get them